sound/soc/sof/ipc4-mtrace.c

```diff
@@ -79,7 +77,7 @@ struct sof_log_state_info {
 	u32 aging_timer_period;
 	u32 fifo_full_timer_period;
 	u32 enable;
-	u32 logs_priorities_mask[MAX_ALLOWED_LIBRARIES];
+	u32 *logs_priorities_mask;
```

Comment:
what are we gaining with this change?
We had cases when the FW_CONFIG data did not had the max_libs_count set, so we fix it up to 1 but that does not mean that the maximum lib count is less than 16.
Or we have different (higher) number in ACE platforms?

Reply:
o, I didn't realize this issue. Pierre asked this quesiton: https://github.com/thesofproject/linux/pull/4287#discussion_r1163368881